src/drivers/imu/mpu6000/mpu6000.cpp

```diff
@@ -2020,11 +2020,16 @@ MPU6000::measure()
 	arb.scaling = _accel_range_scale;
 	arb.range_m_s2 = _accel_range_m_s2;
 
+	// We calculate sensor temperature as :
+	// temperature_real = temperature_raw * temperature_sensitivity + zero_offset
+	//
+	// Sensitivity (in degC/LSB) is from the Invensense datasheet for the specific sensor
```

Comment:
Should the units of sensitivity not be LSB/degC?

Reply:
In the datasheet, yes. However, I defined it here as 1/(LSB/degC). Will make it clearer. Thanks.